package.json

```diff
@@ -12,6 +12,7 @@
     "bignumber.js": "^4.0.2",
     "ethereumjs-tx": "^1.3.1",
     "ethereumjs-util": "^5.1.2",
+    "ethjs-util": "^0.1.4",
```

Comment:
oh, i see the problem... there are two versions of ethUtil being used by the wallet library. my fault, i will fix.